### PR TITLE
Remove: Remove env-logger from navigator-rs library

### PR DESCRIPTION
**Pull request by RaulTrombin**
Due some issues on Jupyter and navigator-assistant,
Noticed that Logger call panick if there is another instances or logging services.
https://docs.rs/env_logger/latest/env_logger/fn.init.html

`UPDATE`
We removed env-logger from navigator-rs library, loggers should be initiallized from the application side.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -384,6 +382,11 @@ impl Navigator {
         self.led.set_led_all(false);
     }
 
+    /// Initialize the log service. Which can display feedback for some processes.
+    pub fn init_logger(&mut self) {
```
Comment (patrickelectric) [inline src/lib.rs:386]: This is not a mutt self and also is not returning.

Comment (RaulTrombin) [reply src/lib.rs:386]: Actually it works like follows:
```
let mut nav = Navigator::new();
nav.init_logger();
nav.init();
nav.set_pwm_freq_hz(60.0);
```

You suggest to:
use a separate `Navigator::init_logger()` 
 
 ```
let mut nav = Navigator::new();
Navigator::init_logger();
nav.init();
nav.set_pwm_freq_hz(60.0);
```

Comment (RaulTrombin) [reply src/lib.rs:386]: oh, just &self.
working on it.

raspberry-pi
Creating your navigator module!
Setting up your navigator, ahoy!
[2023-09-14T15:10:56Z INFO  navigator_rs] Logger initialized successfully
[2023-09-14T15:10:57Z INFO  navigator_rs] PWM frequency set to 60.00 Hz. Prescaler value: 99


and

raspberry-pi
Creating your navigator module!
Setting up your navigator, ahoy!
[2023-09-14T15:10:56Z INFO  navigator_rs] Logger initialized successfully
[2023-09-14T15:11:45Z ERROR navigator_rs] Failed to initialize logger: attempted to set a logger after the logging system was already initialized.
[2023-09-14T15:10:57Z INFO  navigator_rs] PWM frequency set to 60.00 Hz. Prescaler value: 99

**Review by joaoantoniocardoso — CHANGES_REQUESTED** [review]
Actually, I'd rather not use `env-logger`, but only the `log` one, and let the log part always be done on the application side, as mentioned [here](https://github.com/rust-lang/log#in-libraries).

**Comment by RaulTrombin** [comment]
> Actually, I'd rather not use `env-logger`, but only the `log` one, and let the log part always be done on the application side, as mentioned [here](https://github.com/rust-lang/log#in-libraries).

Ok, I agree with the idea,
So we can just remove the entire logger.

**Review by patrickelectric — APPROVED** [review]

**Comment by patrickelectric** [comment]
Will wait @joaoantoniocardoso to approve and merge

**Review by joaoantoniocardoso — APPROVED** [review]

**Pull request merged**